genesis/src/main.rs

```diff
@@ -201,6 +213,11 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                     bootstrap_leader_stake_lamports,
                 ),
             ),
+            // storage account
+            (
+                bootstrap_storage_keypair.pubkey(),
+                Account::new(1, 1024 * 4, &system_program::id()),
```

Comment:
please drive this with a parameter

Reply:
Which part?

Reply:
`1024 * 4`

Reply:
I figured it was for whenever we address that todo.

Reply:
yeah but now that `1024 * 4` is split across 3 random files.  Time to take the bullet I think :)

Reply:
In this PR? master is broken-ish.

Reply:
Can I follow up with another PR for that @rob-solana ? Will be afk for most of this evening.

Reply:
wfm